tools/kubernetes/helm/hue/templates/prometheus-alert-rule.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.alertrule.enabled -}}
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: example
+    role: alert-rules
+  name: prometheus-example-rules
+spec:
+  groups:
+    - name: hue-pod-monitor.alert.rules
+      rules:
+      - alert: hue_pods_are_down
+        expr: sum(up{job="hue",namespace="data-platform-staging"}) < 2 #Based on the replicas mentioned in values.yaml
```

Comment:
Namespace "data-platform-staging", is this one hardcoded and should be extracted to a variable?